tests/integ/modin/groupby/test_groupby_default2pandas.py

```diff
@@ -160,17 +137,93 @@ def test_groupby_with_external_series(basic_snowpark_pandas_df) -> None:
+    "func, kwargs, error_pattern",
+    # note that we expect some functions like `any` to become strings like
+    # 'any' in the error message because of preprocessing in the modin API
+    # layer in [1]. That's okay.
+    # [1] https://github.com/snowflakedb/snowpark-python/blob/7c854cb30df2383042d7899526d5237a44f9fdaf/src/snowflake/snowpark/modin/pandas/utils.py#L633
+    [
+        param(
+            sensitive_function_name,
+            {},
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation Callable with the given arguments",
+            id="user_defined_function",
+        ),
+        param(
+            [sensitive_function_name, "size"],
+            {},
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation \\[Callable, 'size'\\] with the given arguments",
+            id="list_with_user_defined_function_and_string",
+        ),
+        param(
+            (sensitive_function_name, "size"),
+            {},
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation \\[Callable, 'size'\\] with the given arguments",
+            id="tuple_with_user_defined_function_and_string",
+        ),
+        param(
+            {sensitive_function_name, "size"},
+            {},
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation \\[Callable, 'size'\\]|\\['size', Callable\\] with the given arguments",
+            id="set_with_user_defined_function_and_string",
+        ),
+        param(
+            (all, any, len, list, min, max, set, str, tuple, native_pd.Series.sum),
+            {},
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation "
+            + "\\['all', 'any', <built-in function len>, list, 'min', 'max', set, str, tuple, Callable]"
+            + " with the given arguments",
+            id="tuple_with_builtins_and_native_pandas_function",
+        ),
+        param(
+            {
+                "col2": sensitive_function_name,
+                "col3": sum,
+                "col4": "size",
+                "col5": [np.mean, "size"],
+            },
+            {},
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation "
+            + "{label: Callable, label: 'sum', label: 'size', label: \\[Callable, 'size'\\]}"
+            + " with the given arguments",
+            id="dict",
+        ),
+        param(
+            None,
+            {
+                "new_col": ("col2", sensitive_function_name),
+                "new_col2": pd.NamedAgg("col3", sum),
+            },
+            "Snowpark pandas GroupBy.aggregate does not yet support the aggregation "
+            + "new_label=\\(label, Callable\\), new_label=\\(label, <built-in function sum>\\)"
+            + " with the given arguments",
+            id="named_agg",
+        ),
+    ],
+)
+@sql_count_checker(query_count=0)
+def test_groupby_agg_unsupported_function_SNOW_1305464(
```

Comment:
Shouldn't we also add back the test for a function that is unsupported, e.g. np.sin?

Reply:
Done with `np.argmin`.